frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -167,17 +167,17 @@ export const surveyLogic = kea<surveyLogicType>([
                 ) => {
                     const question = isEditingQuestion
                         ? state.questions[idx].question
-                        : defaultSurveyFieldValues[type].questions[idx].question
+                        : defaultSurveyFieldValues[type].questions[0].question
```

Comment:
This is confusing, there's no reason for `defaultSurveyFieldValues` to have a questions array anymore. Worth a follow up refactor.